cmiles/tests/reference/generate_reference.py

```diff
@@ -0,0 +1,47 @@
+import cmiles
+from cmiles.tests import utils
+import rdkit
+version = rdkit.__version__
+print(version)
+# Regenerate with molecule names (makes it easier to search later)
+
+def get_smiles_list(f):
+    smiles = open(f, 'r').read().split('\n')[:-1]
+    names = []
+    for i in range(len(smiles)):
+        smiles_name = smiles[i].split(' ')
+        smiles[i] = smiles_name[0]
+        names.append(smiles_name[-1])
+    return smiles, names
+#smiles, name = get_smiles_list('drug_bank_stereo.smi')
```

Comment:
Is it safe to clean up unused code in this file (like `get_smiles_list`)? And adding a small docstring covering the purpose of this file would be helpful for when I come back to this after you're gone.

Reply:
I removed the unused code. It's not needed and unimportant.